provider/blob/src/blob_schema.rs

```diff
@@ -39,6 +39,14 @@ impl<'data> BlobSchema<'data> {
         }
     }
 
+    #[cfg(feature = "export")]
```

Comment:
Discussion: I think `"export"` isn't really the right feature. You can be using BlobDataProvider as the datagen source without enabling the export feature. Maybe name the feature just `"datagen"`?

Reply:
> You can be using BlobDataProvider as the datagen source without enabling the export feature

You really can't, because icu_datagen enables the export feature. I don't want to complicate datagen features even more by having two blob features.